src/azure-cli/azure/cli/command_modules/eventgrid/_params.py

```diff
@@ -18,9 +18,11 @@
 )
 
 from .advanced_filter import EventSubscriptionAddFilter
+from .event_channel_filter import EventChannelAddFilter
+from .inbound_ip_rules import AddInboundIpRule
 
 included_event_types_type = CLIArgumentType(
-    help="A space-separated list of event types (e.g., Microsoft.Storage.BlobCreated Microsoft.Storage.BlobDeleted). To subscribe to all default event types, do not specify this argument. For event grid topics, event types are customer defined. For Azure events, e.g., Storage Accounts, IoT Hub, etc., you can query their event types using this CLI command 'az eventgrid topic-type list-event-types'.",
+    help="A space-separated list of event types (e.g., Microsoft.Storage.BlobCreated and Microsoft.Storage.BlobDeleted). In order to subscribe to all default event types, do not specify any value for this argument. For event grid topics, event types are customer defined. For Azure events, e.g., Storage Accounts, IoT Hub, etc., you can query their event types using this CLI command 'az eventgrid topic-type list-event-types'.",
```

Comment:
Is there a way to get all related types?

Reply:
The help text is helping user to get this info specifically:

you can query their event types using this CLI command 'az eventgrid topic-type list-event-types'.